src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/manageCaseAssignments/casesharing.scala

```diff
@@ -0,0 +1,259 @@
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .check(status is 200)
+        .check(jsonPath("$.access_token").saveAs("access_tokenShare")))
+
+  //  .exec {
+  //      session =>
+  //        println(session("bearerToken").as[String])
+  //        println(session("access_token").as[String])
+  //        session
+  //    }
+
+  val CaseShareLoginSmall =
+
+    exec(http("02_ACA_GetS2SToken")
+      .post("http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal/testing-support/lease")
+      .header("Content-Type", "application/json")
+      .body(StringBody("{\"microservice\":\"xui_webapp\"}"))
+      .check(bodyString.saveAs("bearerTokenShare")))
+
+      //.feed(feedCSUserDataSmall)
+
+      .exec(http("02_ACA_OIDC01_Authenticate")
+        .post(Environment.idamAPI + "/authenticate")
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .formParam("username", "${email}")
+        .formParam("password", "${password}")
+        .formParam("redirectUri", Environment.xuiMCUrl)
+        .formParam("originIp", "0:0:0:0:0:0:0:1")
+        .check(status is 200)
+        .check(headerRegex("Set-Cookie", "Idam.Session=(.*)").saveAs("authCookie")))
+
+      .exec(http("02_ACA_OIDC02_Authorize_XUI")
+        .post(Environment.idamAPI + "/o/authorize?response_type=code&client_id=" + xuiClientId + "&redirect_uri=" + Environment.xuiMCUrl + "&scope=" + xuiScope).disableFollowRedirect
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .header("Cookie", "Idam.Session=${authCookie}")
+        .header("Content-Length", "0")
+        .check(status is 302)
+        .check(headerRegex("Location", "code=(.*)&client_id").saveAs("code")))
+
+      .exec(http("02_ACA_OIDC03_Token_XUI")
+        .post(Environment.idamAPI + "/o/token?grant_type=authorization_code&code=${code}&client_id=" + xuiClientId + "&redirect_uri=" + Environment.xuiMCUrl + "&client_secret=" + xuiwebappClientSecret)
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .check(status is 200)
+        .check(jsonPath("$.access_token").saveAs("access_tokenShare")))
+
+  //  .exec {
+  //      session =>
+  //        println(session("bearerToken").as[String])
+  //        println(session("access_token").as[String])
+  //        session
+  //    }
+
+  val CaseSharingPostLarge =
+
+    exec(http("02_CS_010_AssignCaseLargeOrg")
+      .post(mcaUrl + "/case-assignments")
+      .header("ServiceAuthorization", "Bearer ${bearerTokenShare}")
+      .header("Authorization", "Bearer ${access_tokenShare}")
+      .header("Content-Type", "application/json")
+      .header("Accept", "application/json")
+      .body(StringBody("{\"assignee_id\":\"${assigneeUser}\",\"case_id\":\"${caseToShare}\",\"case_type_id\":\"GrantOfRepresentation\"}")))
+
+      .pause(Environment.constantthinkTime)
+
+  val CaseSharingPostSmall =
```

Comment:
As above, do you need all this duplicated code, or can you just make small/large a variable that you pass as an argument?

Reply:
this code is existing in the repo and not mine - the casesharing.scala file has only been moved into a new folder.

Reply:
I have now removed this file from the repo as the code is being rewritten as part of the manage case PI28